_episodes/12-scheduler.md

```diff
@@ -165,12 +170,18 @@ which is probably not what we want.
 
 The following are several key resource requests:
 
+* `-n <nnodes>` - how many nodes does your job need? 
```

Comment:
I'll admit I didn't want to get too far into multi-node jobs for these intro materials... that said, `-n` specifies number of tasks which does not necessarily correspond to number of nodes if `-c` is set (cpus per task). Could you add a bit of clarification here on this and then I'll merge this pull request?

Reply:
I was lazy and didn't look up the right option for nodes ( `-N`).  Oops. 

I was thinking bigger picture -- having a list of the common options that people should know about (whether or not they need them immediately).

Reply:
Ok sounds good - I will add some clarification and merge this in (I might move the an overview of the differences between `-n` and `-N` into a callout box somewhere).